Source/WebCore/page/WorkerNavigator.cpp

```diff
@@ -66,6 +68,15 @@ void WorkerNavigator::setAppBadge(std::optional<unsigned long long> badge, Ref<D
         return;
     }
 
+#if ENABLE(DECLARATIVE_WEB_PUSH)
+    if (is<ServiceWorkerGlobalScope>(*scope)) {
+        if (auto* pushNotificationEvent = downcast<ServiceWorkerGlobalScope>(*scope).pushNotificationEvent()) {
```

Comment:
RefPtr

Reply:
For the same reason as the last one about the current notification even, not needed.

Reply:
Please, no raw pointer variable in new code, even if you think it is safe.